src/citrine/informatics/predictors/predictor.py

```diff
@@ -1,125 +0,0 @@
-import warnings
-from typing import Optional, Type, List
-from uuid import UUID
-
-from citrine._rest.asynchronous_object import AsynchronousObject
-from citrine._serialization import properties
-from citrine._serialization.polymorphic_serializable import PolymorphicSerializable
-from citrine._session import Session
-from citrine.informatics.data_sources import DataSource
-from citrine.resources.report import ReportResource
-from citrine.informatics.predictors.single_predict_request import SinglePredictRequest
-from citrine.informatics.predictors.single_prediction import SinglePrediction
-from citrine._utils.functions import format_escaped_url
-
-__all__ = ['Predictor']
-
-
-class Predictor(PolymorphicSerializable['Predictor'], AsynchronousObject):
```

Comment:
The `GraphPredictor` assumes all fields from this interface, since there is only one type.

We can change the inheritance chain back to maintain backwards compat if we deem. necessary.